### ISSUE TITLE
php-xdebug not available for PHP7.2

### ISSUE THREAD

**Issue by ssaki** [oerdnj/deb.sury.org#751]
Hi,

it seems that XDebug is still unavailable for PHP 7.2.

```
# apt-cache policy php-xdebug 
php-xdebug:
  Installed: 2.5.5-3+ubuntu16.04.1+deb.sury.org+1
  Candidate: 2.5.5-3+ubuntu16.04.1+deb.sury.org+1
  Version table:
 *** 2.5.5-3+ubuntu16.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
     2.4.0-1 500
        500 http://mirror.telepoint.bg/ubuntu xenial/universe amd64 Packages
```

```
 # apt-cache depends php-xdebug
php-xdebug
  Depends: ucf
  Depends: php-common
 |Depends: <phpapi-20160303>
    libapache2-mod-php7.1
    libphp7.1-embed
    php7.1-cgi
    php7.1-cli
    php7.1-fpm
    php7.1-phpdbg
 |Depends: <phpapi-20151012>
    libapache2-mod-php7.0
    libphp7.0-embed
    php7.0-cgi
    php7.0-cli
    php7.0-fpm
    php7.0-phpdbg
  Depends: <phpapi-20131226>
    libapache2-mod-php5.6
    libphp5.6-embed
    php5.6-cgi
    php5.6-cli
    php5.6-fpm
    php5.6-phpdbg
  Depends: libc6
```
Kind regards

**Comment by VolCh** [comment]
Also php-amqp not available

**Comment by oerdnj** [comment]
Yes, because the last time I checked it failed to build.

**Comment by oerdnj** [comment]
@VolCh Please open a new issue. Don't attach random comments to random issues...

**Comment by morloderex** [comment]
@oerdnj xdebug is now on version 2.6alpha release. Which supports 7.2.

**Comment by oerdnj** [comment]
Great, I'll wait for xdebug 2.6 final, I am not uploading alpha release just to support PHP 7.2 that was released couple of days ago.

**Comment by oerdnj** [comment]
@morloderex I know that you are eager to get the xdebug for PHP 7.2, but do you really think that leaving out this important information would benefit anyone?
> This is not a final release, and it does have known issues, including a crash bug.

**Comment by morloderex** [comment]
@oerdnj It was just a heads up. Nothing else. I'll wait or compile it myself if needed. 
Could you please give us a status on #596?

**Comment by swiffer** [comment]
Based on dericks reply in Twitter it's Just Alpha because its Not Feature complete. https://twitter.com/DracoBlue/status/940962822707732483
Would be great to have it available for Testing

**Comment by oerdnj** [comment]
That would require jumping through hoops to have 2.6.0 just for PHP 7.2.0. If you want to test it, use PECL for that.

**Comment by swiffer** [comment]
Ok, will do so -> meanwhile: thanks for your effort continously releasing new packages

**Comment by oerdnj** [comment]
thanks for understanding

**Comment by pavlakis** [comment]
I wrote blog post on [Setting up Xdebug with Docker and PHP 7.2](https://medium.com/@pavlakis/setting-up-xdebug-with-docker-and-php-7-2-f5a6553cb956) for anyone who might find it useful. 
I know this thread is not Docker specific, but some of the steps are the same.

**Comment by morloderex** [comment]
@oerdnj can you update your repository now that xdebug 2.6 is stable?

**Comment by luispabon** [comment]
would be great, waiting on this to upgrade my current proj to 7.2

**Comment by luispabon** [comment]
@oerdnj I've been testing the xdebug 2.6.0 build on the QA repository for 17.10:

`2.6.0+2.5.5-1+ubuntu17.10.1+deb.sury.org+1`

I have noticed a very marked drop in performance as compared with 2.5.5 + PHP 7.1 from the normal PPA. 

My current project has a small amount of end to end phpunit tests on a symfony 4 app. On PHP 7.1 + xdebug 2.5.5 (both from the stable ppa for 17.10) these tests take an average of ~500ms each, whereas the very same tests on the same machine (php 7.2 from stable ppa + xdebug 2.6.0 from qa ppa) take around 800ms each.

Is this something to be expected because of the way packages are built on the qa repo (eg with debug on etc)?

**Comment by thexpand** [comment]
Any progress on this? It seems that 2.6.0 has been released.
https://xdebug.org/updates.php

**Comment by oerdnj** [comment]
xdebug 2.6.0 for PHP 7.x and xdebug 2.5.x for PHP 5.x has been published in the PPA.

**Comment by oerdnj** [comment]
@luispabon No, the QA repo is not different from the main. I just land packages that might break something first - I had to repackage xdebug to support PHP 5.x.

**Comment by patrickbussmann** [comment]
But in your repository it isnt available, or?

```
$ apt-cache search xdebug
php-xdebug - Xdebug Module for PHP
```

And `php --info` not find it, too.

```
$ php --info | grep debug
report_zend_debug => Off => Off
opcache.opt_debug_level => 0 => 0
```

Where I can find it?
